File: Data Structures & Algorithms/sliding-window-maximum/submission-1.java
class Solution {
    public int[] maxSlidingWindow(int[] nums, int k) {
        if(nums==null||nums.length==0){
            return new int[0];
        }

        int left=0;
        int right=left+k;
        int[] arr = new int[nums.length - k + 1];
        while(right<=nums.length){
            int max=nums[left];
            for(int i=left; i<left+k; i++){
                if(nums[i]>max){
                    max=nums[i];
                }
            }
            arr[left]=max;
            left++;
            right++;
        }
        return arr;
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        right=len(numbers)-1
        left=0
        while left<right:
            if numbers[right-1]+numbers[right]<target:
                return []
            elif numbers[left]+numbers[right]>target:
                right-=1
            elif numbers[left]+numbers[right]<target:
                left+=1
            else:
                return [left+1,right+1]

            
            